notebooks/spark/AerospikeSparkScala.ipynb

```diff
@@ -6,35 +6,81 @@
    "source": [
     "# Aerospike Spark Connector Tutorial for Scala\n",
     "\n",
-    "## Tested with Spark connector 3.0.1, Java 8, Apache Spark 3.0.0, Python 3.7  and Scala 2.12.11 and Spylon ( https://pypi.org/project/spylon-kernel/)\n"
+    "## Tested with Spark connector 2.8.0, Java 8, Apache Spark 2.4.0, Python 3.7  and Scala 2.11.12 and Spylon ( https://pypi.org/project/spylon-kernel/)\n"
    ]
```

Comment:
general comment: remove dead/unused code, user hyperlinks (not links) and point to the docs wherever you can without duplicating information.